address-space-controller/pom.xml

```diff
@@ -59,6 +59,11 @@
       <artifactId>slf4j-api</artifactId>
       <scope>compile</scope>
     </dependency>
+    <dependency>
+      <groupId>io.fabric8</groupId>
+      <artifactId>openshift-server-mock</artifactId>
```

Comment:
Much cleaner test code resulting from this - excellent

Reply:
Yeah, openshift-server-mock came with the latest kubernetes-client release, as the previous versions only had kubernetes-server-mock.